test/gui/shared/scripts/bdd_hooks.py

```diff
@@ -76,11 +78,11 @@ def hook(context):
     # try reading configs from config.ini
     cfg = ConfigParser()
     try:
-        cfg.read('../config.ini')
-        for key, value in context.userData.items():
-            if value == '':
-                context.userData[key] = cfg.get('DEFAULT', CONFIG_ENV_MAP[key])
-    except Exception as err:
+        if cfg.read('../config.ini'):
+            for key, value in context.userData.items():
+                if value == '':
+                    context.userData[key] = cfg.get('DEFAULT', CONFIG_ENV_MAP[key])
```

Comment:
Don't try to check for values if `cfg.read()` returns empty.
This prevents logs like `No option 'max_sync_timeout' in section: 'DEFAULT'`